src/test/java/uk/gov/hmcts/sptribs/common/notification/CaseWithdrawnNotificationTest.java

```diff
@@ -26,6 +25,9 @@
 import static org.mockito.Mockito.doNothing;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.sptribs.caseworker.model.CloseReason.DeathOfAppellant;
+import static uk.gov.hmcts.sptribs.caseworker.model.CloseReason.Withdrawn;
+import static uk.gov.hmcts.sptribs.common.CommonConstants.DEATH_OF_APPELLANT_EMAIL_CONTENT;
```

Comment:
Only a small formatting thing - is this import really needed when CommonConstants is already imported? The others are called via CommonConstants.CONSTANT_NAME - for consistency should this not be the same?

Reply:
Sure can use static imports for all common constants